assets/javascripts/discourse/initializers/discourse-brightcove.js.es6

```diff
@@ -105,7 +106,7 @@ function initializeBrightcove(api) {
           bootbox.alert(
             I18n.t("brightcove.not_allowed", {
               trust_level: siteSettings.brightcove_min_trust_level,
-              trust_level_description: Discourse.Site.currentProp("trustLevels")
+              trust_level_description: Site.currentProp("trustLevels")
```

Comment:
Great change but it would be even better if we could look up site on the container!

Reply:
Ha, that's how I did it in the first place, but then noticed we're doing `import Site` in core in a lot of places, so I force-pushed this version. 😃

What's the officially recommended way? `this.site` in controllers, components, routes, models, and adapters, and `api.container.lookup("site:main")` in plugin initializers?

Reply:
The goal should really be to avoid using the Singleton `Site` anywhere. Same with `User`! I know it's a huge amount of work though so I don't expect us to get there any time soon. It's much better if we are referring to object instances in the container than a global module.